code-challenges/401/linkedList/lib/ll.js

```diff
@@ -191,6 +192,35 @@ class LinkedList {
     return outputString;
   }
 
+
+}
+
+
+function mergeLists(l1, l2){
+  
+  
+  let list1Current = l1.head;
+  let list2Current = l2.head;
+  let list1Next = new Node();
+  let list2Next = new Node();
+
+  // console.log(first, second);
+
+  while (list1Current !== null || list2Current !== null){
+    
+    list1Next = list1Current.next;
+    list2Next = list2Current.next;
+
+    list1Current.next = list2Current;
+    list2Current.next = list1Next;
+
+    list1Current = list1Next;
+    list2Current = list2Next;
```

Comment:
You can add some if conditionals here that say if this is null, then next is sent to the other list
   - I would do this for both lists